test.js

```diff
@@ -1,8 +1,8 @@
 import test from 'ava';
-import delay from 'delay';
+import delay from 'yoctodelay'; // TODO: Replace with `import {setTimeout as delay} = from 'timers/promises';` when targeting Node.js 16
 import inRange from 'in-range';
 import timeSpan from 'time-span';
-import pDebounce from '.';
+import pDebounce from './index.js'; // eslint-disable-line unicorn/import-index, import/extensions
```

Comment:
Just doing `import pDebounce from '.';` throws `ERR_UNSUPPORTED_DIR_IMPORT` because Node.js [can't import](https://stackoverflow.com/a/64507978/8384910) the directory: `'.'`. Adding the full name works but XO throws `unicorn/import-index` and `import/extensions`.